common/static/common/templates/components/paging-footer.underscore

```diff
@@ -1,16 +1,16 @@
-<nav class="pagination pagination-full bottom">
+<nav class="pagination pagination-full bottom" aria-label="Full Pagination">
 <ol>
-    <li class="nav-item previous"><a class="nav-link previous-page-link" href="#"><i class="icon fa fa-angle-left"></i> <span class="nav-label"><%= gettext("Previous") %></span></a></li>
+    <li class="nav-item previous"><a class="nav-link previous-page-link" href="#"><i class="icon fa fa-angle-left" aria-hidden="true"></i> <span class="nav-label"><%= gettext("Previous") %></span></a></li>
```

Comment:
@dan-f I think that if we are going to use ```<button>```s in the compact nav, we should use ```<button>```s here as well.  ```href="#"``` will actually fail validation tests.  I also do not think we need to structure this as a list.  I try to restrict using lists, ordered or unordered, for lists of items, not for a structure to hang CSS on.  an ordered list implies that the items need to be completed in order, and that wouldn't be the case here.